src/frontend/src/handler/create_sql_function.rs

```diff
@@ -53,6 +59,23 @@ fn create_mock_udf_context(
     ret
 }
 
+/// Find the pattern for better hint display
+/// return the exact index where the pattern first appears
+fn find_target(input: &str, target: &str) -> Option<usize> {
+    // Regex pattern to find `target` not preceded/followed by an ASCII letter.
+    // \b asserts a word boundary, and \B asserts NOT a word boundary
```

Comment:
What's the comment for? (So why not use `\b`?)

Reply:
Oh I forgot to change it back to `\b` after trying negative / positive lookahead 😅.